packages/langchain/README.md

```diff
@@ -0,0 +1,122 @@
+# @sap-ai-sdk/langchain
+
+This package contains Langchain compliant models, based on the @sap-ai-sdk clients.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/langchain
+
+$ npm install @langchain/openai // if you want to use OpenAI models
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
+- Project configured with Node.js v20 or higher and native ESM support enabled.
+- For testing your application locally:
+  - Download a service key for your AI Core service instance.
+  - Create a `.env` file in the sample-code directory.
+  - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
+
+## Usage
+
+All client's comply with [Langchain's interface](https://python.langchain.com/v0.2/api_reference/openai/chat_models/langchain_openai.chat_models.azure.AzureChatOpenAI.html#langchain_openai.chat_models.azure.AzureChatOpenAI), therefore you should be able to use them as per usual.
+
+The only difference is in the initialization of the client, where you have th option to pass either:
+
+```ts
+    modelName: string,
+    modelVersion?: string,
+    resourceGroup?: string,
+    ...others
+```
+
+or
+
+```ts
+    deploymentId: string,
+    resourceGroup?: string
+    ...others
+```
+
+If you pass API keys they are ignored, since you're not inteded to call the vendor's endpoints directly.
+Instead, the credentials in the binding are used to call SAP's LLM Proxy.
```

Comment:
[q] is this the official name?

Reply:
I wasn't sure what the name of the component we call with the credentials is, do you have a better suggestion? 😬

Reply:
```suggestion
Instead, credentials in the binding are required to access SAP AI Core capabilities, specifically for consuming LLMs via the SAP Generative AI Hub.
```